packages/frontend-shared/cypress/support/component.ts

```diff
@@ -2,7 +2,10 @@ import { registerMountFn, addVueCommand } from './common'
 import '../../src/styles/shared.scss'
 import 'virtual:windi.css'
 import installCustomPercyCommand from '@packages/ui-components/cypress/support/customPercyCommand'
+import { goOffline, goOnline } from './onlineNetwork'
 
 registerMountFn()
 addVueCommand()
 installCustomPercyCommand()
+goOffline()
```

Comment:
Can we rename these to something like `addGoOffline` and `addGoOnline` or even `addNetworkCommands`? Just reading this file, it looks like we actually go offline, then go back online, at the start of each component test.